infra/base-images/base-builder/jcc/jcc.go

```diff
@@ -391,6 +403,7 @@ func main() {
 		//  to make build systems happy.
 		fmt.Print(out)
 		fmt.Fprint(os.Stderr, errstr)
+               appendStringToFile("/out/err.log", errstr)
```

Comment:
You may want to do this everywhere there is an Fprintf(os.stderr).

Reply:
Fixed, thanks!